src/main/java/com/filip/WorkshopManagment/Models/Customer.java

```diff
@@ -13,30 +15,19 @@
     private String mobile;
     private String email;
     private String notes;
-    @OneToMany
-    private List<Vehicles> vehicles;
```

Comment:
...
@OneToMany(mappedBy = "customer")
@JoinColumn(name="customer_id")
private List<Vehicles> vehicles;
...
To chyba jakoś tak powinno wyglądać